file.go

```diff
@@ -594,6 +594,10 @@ type ValidateOpts struct {
 	// BypassCompanyIdentificationMatch allows batches in which the Company Identification field
 	// in the batch header and control do not match.
 	BypassCompanyIdentificationMatch bool `json:"bypassCompanyIdentificationMatch"`
+
+	// CustomReturnCodes can be set to skip validation for the Return Code field in an Addenda99
+	// This allows for non-standard/deprecated return codes (e.g. R97)
+	CustomReturnCodes bool `json:"customReturnCodes"`
```

Comment:
@vxio What are your thoughts on a `CheckReturnCode func(code string) error` function that allows validating codes like "R97" as the issue mentions.

Reply:
For ref: https://moov-io.slack.com/archives/CD9J8EJKX/p1626989492134600?thread_ts=1626981488.133100&cid=CD9J8EJKX

I went with the bool so that I can call from an API endpoint more easily with the base docker image

Reply:
Yea that's a good idea. We can add a validation func in a separate PR.